### ISSUE TITLE
Feature Request: Changing bot playlist

### ISSUE THREAD

**Issue by bloodhoundx** [samuri51/chillybot#60]
Is there a way to implement this feature?

**Comment by samuri51** [comment]
yep, the ttapi provides the playlistSwitch function to do exactly this

playlistSwitch ( playlistName:string [ , callabck:fn ] )
Switch to another playlist.

**Comment by bloodhoundx** [comment]
Thanks for the quick reply, I will look into it!
